samples/bluetooth/fast_pair/locator_tag/src/ui/platform_dk.c

```diff
@@ -156,39 +163,27 @@ static void mode_led_thread_process(void)
 	static bool mode_led_on;
 
 	while (1) {
+		bool id_mode = atomic_test_bit(ui_state_status, APP_UI_STATE_ID_MODE);
+		bool recovery_mode = atomic_test_bit(ui_state_status, APP_UI_STATE_RECOVERY_MODE);
+		bool both_modes = recovery_mode && id_mode;
```

Comment:
Can we check modes explicitly here (remove `both_modes`) to improve consistency with provisioning LED? Alternatively, we could add something similar to `bool both modes` to provisioning LED.